frontend/src/components/Notification.js

```diff
@@ -47,28 +96,31 @@ class Notification extends Component {
   render = () => {
     const { notification } = this.props
     const classes = `notification ${notification.is_read ? 'read' : 'unread'}`
+
+    if (!notification.data.object) {
+      return null
+    }
+
     return <li className={classes}>
       <a href={`/notifications/${notification.id}`}>
         <div className='notification-actor'>
           <img src={notification.actor.image} />
         </div>
-        <div className="notification-body">
-          <div className="in-bold">{notification.actor.name}</div>
-          Other content
-        </div>
+        <div className='notification-body'
+             dangerouslySetInnerHTML={this.notificationTextHtml()} />
       </a>
-      <div className="notification-read-unread">
+      <div className='notification-read-unread'>
         {!notification.is_read && <div onClick={this.markAsRead}>
           mark read
         </div>}
         {notification.is_read && <div onClick={this.markAsUnread}>
           mark unread
         </div>}
       </div>
-      <div className="notification-date">
+      <div className='notification-date'>
         {this.getDate()}
       </div>
-      <div className="clearfloat"></div>
+      <div className='clearfloat'></div>
```

Comment:
I don't view this as blocking, but I think JSX props should use double quotes by default. It's a more common style, and it more closely mirrors HTML